lib/commands/context/helpers.js

```diff
@@ -738,6 +738,17 @@ export async function setupNewChromedriver(opts, curDeviceId, context) {
     }
   }
 
+  // Ensure there are logging preferences
+  opts.chromeLoggingPrefs = opts.chromeLoggingPrefs || {};
+
+  // Strip the prefix and store it
+  for (const opt of _.keys(opts)) {
+    if (opt.endsWith(':loggingPrefs')) {
+      this?.log?.warn(`Merging '${opt}' into 'chromeLoggingPrefs'. This may cause unexpected behavior`);
```

Comment:
these optional chains are redundant as both properties are expected to be always defined

Reply:
I am afraid I do not understand. Can you help explain please?